tutorretirement/templates/retirement/build/retirement/pipeline_config/config.yml

```diff
@@ -3,13 +3,11 @@ client_secret: {{ RETIREMENT_EDX_OAUTH2_CLIENT_SECRET }}
 
 base_urls:
     lms: {{ "https" if ENABLE_HTTPS else "http" }}://{{ LMS_HOST }}
-    {% if ECOMMERCE_HOST is defined %}ecommerce: {{ "https" if ENABLE_HTTPS else "http" }}://{{ ECOMMERCE_HOST }}{% endif %}
```

Comment:
I'm not sure how important this is to the community, but if you remove this it may impact the ability to retire all of the Segment IDs. I've got doubts if anyone in the community is using ecommerce + segment + tutor, but thought I'd mention it.

tl;dr: ecommerce can maintain a separate user id just for interacting with Segment. If the retirement config sets `fetch_ecommerce_segment_id` to True and includes the Segment retirement step, the scripts will query Ecommerce for this id and the segment_api will retire it along with the other things that get deleted from Segment. This is what the ecommerce retirement API that we were looking at in the other issue handles.

- https://github.com/openedx/edx-platform/blob/6d7fd926d151017d4fd2a502d7f47bc505f3e752/scripts/user_retirement/retire_one_learner.py#L176
- https://github.com/openedx/edx-platform/blob/6d7fd926d151017d4fd2a502d7f47bc505f3e752/scripts/user_retirement/utils/thirdparty_apis/segment_api.py#L18

Reply:
@bmtcril We maintain a community Tutor plugin here. If ecommerce is, as you say, in such a state that a reasonable Open edX community deployment shouldn't be using it at all, and the retirement pipeline for it is in such a state that there is effectively no retirement for ecommerce at all, then I don't see any impetus for a community plugin to support it. Much less support something related to the integration of that thing with a third-party service.

I'd much prefer we explicitly *not* support it, so that people who are brave (reckless?) enough to use ecommerce (with or without Segment) know that they're on their own as far as retirement for those services is concerned.

Reply:
Sounds good to me